cmd/porter/bundle.go

```diff
@@ -31,14 +31,23 @@ func buildBundleCommands(p *porter.Porter) *cobra.Command {
 }
 
 func buildBundleCreateCommand(p *porter.Porter) *cobra.Command {
-	return &cobra.Command{
-		Use:   "create",
+	cmd := &cobra.Command{
+		Use:   "create [bundle-name]",
 		Short: "Create a bundle",
-		Long:  "Create a bundle. This generates a porter bundle in the current directory.",
+		Long:  "Create a bundle. This generates a porter bundle with the directory with the specified name or in the root directory if no name is provided.",
```

Comment:
I would leave this as "current directory" or use "working directory" instead of "root directory"
Root directory can have a lot of different meanings depending on the users background, and in my interpretation that is not where I would want my bundle. :)